turbinia/pubsub.py

```diff
@@ -127,9 +144,24 @@ def send_message(self, message):
 
     message: The message to send.
     """
-    data = message.encode('utf-8')
-    future = self.publisher.publish(self.topic_path, data)
-    msg_id = future.result()
+    base64_data = base64.b64encode(message.encode('utf-8'))
+    request_body = {
+        "messages": [{
+            "data":
+                base64_data.decode('utf-8')  # base64 encoded string
+        }]
+    }
+    publish_client = self.pubsub_api_client.projects().topics()
+    response = gcp_common.ExecuteRequest(
+        publish_client, 'publish', {
+            'topic': self.topic_path,
+            'body': request_body
+        })
+    # Safe to unpack since response is unpaged.
+    if not response[0]['messageIds']:
+      raise TurbiniaException(
+          f'Message {message} was not published to topic {self.topic_path}')
```

Comment:
Tiny nit, but could you format the string here (and the other couple format strings that are added) with a format string like `{0:s}` just for consistency with the rest of the codebase?  We're not using f'' strings yet, and still have the older style strings as the default in the style guide: https://turbinia.readthedocs.io/en/latest/developer/contributing.html#style-guide